media-art-background.js

```diff
@@ -1,9 +1,10 @@
-function setBackground(root, lovelace, app, originalStyle) {
+function setBackground(root, lovelace, bground_elem, originalStyle) {
   let hass = root.hass;
   for (let entity of lovelace.config.media_art_background.entities) {
     let entityName = entity.entity || entity;
     let entityValidStates = entity.valid_states || ['playing'];
     let entityImageAttribute = entity.image_attribute || 'entity_picture';
+    let filterBlur = lovelace.config.media_art_background.blur || '0px';
```

Comment:
This is a "global" config setting, so please read it before going into the loop, as it's not entity-specific.